src/runtime/test/hydrate-no-encapsulation.spec.tsx

```diff
@@ -166,6 +166,69 @@ describe('hydrate no encapsulation', () => {
     `);
   });
 
+  it('nested text slot with key', async () => {
+    @Component({ tag: 'cmp-a' })
+    class CmpA {
+      render() {
+        return (
+          <Host key="test">
+            <cmp-b key="my-test-key">light-dom</cmp-b>
+          </Host>
+        );
+      }
+    }
+    @Component({ tag: 'cmp-b' })
+    class CmpB {
+      render() {
+        return (
+          <Host>
+            <slot key="key1"></slot>
+            <footer key="key2"></footer>
+          </Host>
+        );
+      }
+    }
+    // @ts-ignore
```

Comment:
For my own knowledge, why do we need this `@ts-ignore`? If we need it, can we add a justification/reason why we need it in the comment for future us?

Reply:
We don't! I copy-pasted an existing test which has this (every test in this file does) and just assumed it was necessary. However, it's not! That's for two reasons:

- we don't type-check the `spec.tsx` files anyway (they're not included in `tsconfig.include`) so removing these doesn't cause any new errors
- it appears that additionally whatever errors these were added to suppress aren't present, since if I add `src/**/*.tsx` to `include` in `tsconfig.json` and then start removing these `@ts-ignore` comments I don't see any new errors

So I'm going to go ahead and just remove them all from the file